kraken-build/src/kraken/std/python/buildsystem/uv.py

```diff
@@ -39,14 +37,17 @@ def find_uv_bin() -> str: ...
 
 logger = logging.getLogger(__name__)
 T = TypeVar("T")
-T_PackageIndex = TypeVar("T_PackageIndex", bound=PackageIndex)
 Safe = Annotated[T, "safe"]
 Unsafe = Annotated[T, "unsafe"]
 
 
 @dataclass
-class PipIndex:
+class UvIndex:
+    # https://docs.astral.sh/uv/configuration/indexes/#defining-an-index
     url: str
+    name: str = ""
```

Comment:
`str | None` ? or do we only encode the named indexes here (and then should avoid an empty string default value ?)

Reply:
I'm updating your suggestion here to represent the optional name with None. However, I decided not to update the `alias` field in the `PackageIndex` dataclass. See my argument bellow: https://github.com/kraken-build/kraken/pull/297#discussion_r1925342139